.github/workflows/test-action.yml

```diff
@@ -43,28 +27,10 @@ jobs:
 
       - name: Trigger Slackbot Channel Test
         id: trigger-slackbot-channeltest
-        uses: ./go/src/slack-bot-action
+        uses: patriotsoftware/slack-bot-action@v1
         with:
           destination: committer
           message: "A new slackbot update has been triggered."
           slack-token: ${{ secrets.SLACK_TOKEN }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
           fallback-destination: "#alerts-devops"
-
-      # Print out the outputs from the previous step
-      - name: Verify Output
-        run: |
-          if [ -z "${{ steps.trigger-slackbot-channeltest.outputs.validate-output }}" ]; then
-            echo "❌ Step did not complete successfully."
-            exit 1
-          fi
-          echo "✅ Action completed"
-          echo "${{ steps.trigger-slackbot-channeltest.outputs.validate-output }}"
-    
-      - name: Auto-commit JS changes
-        uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: Updated slack-bot-action GitHub Action src
-          branch: ${{ steps.extract_branch.outputs.branch }}
-          file_pattern: ${{ env.basefile }}.js* node_modules/*
-          repository: ${{ env.workdir }}
```

Comment:
This test workflow is pretty useless but I'm leaving it for now, adjusted to somewhat validate the publushed release works